tests/jballerina-integration-test/src/test/resources/messaging/artemis/src/transaction/simple_consumer.bal

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/artemis;
+import ballerina/io;
+
+public function createSimpleConsumer() returns artemis:Consumer|error {
+    artemis:Listener lis = new artemis:Listener({host: "localhost", port: 61616});
+    return lis.createAndGetConsumer({queueName: "example"});
+}
+
+public function testSimpleTransactionSend() {
+    artemis:Producer prod = new({host: "localhost", port: 61616}, "example");
+    send2(prod);
+    transaction {
+        send2(prod);
+    }
+}
+
+public function transactionConsumerReceive(artemis:Consumer consumer) returns @tainted string {
+    string msgTxt = "";
+    transaction {
+       msgTxt += receiveAndGetText(consumer);
+       msgTxt += receiveAndGetText(consumer);
+    }
+    return msgTxt;
+}
+
+function send2(artemis:Producer prod) {
+    var err = prod->send("Example ");
+    if (err is error) {
+        io:println("Error occurred sending message");
```

Comment:
We can remove this print statement and panic error to fail the test

Reply:
Resolved